scenariolib/visit.go

```diff
@@ -208,19 +208,19 @@ func (v *Visit) sendClickEvent(rank int, quickview bool, customData map[string]i
 	} else {
 		return errors.New("Cannot convert urihash to string")
 	}
-	if collection, ok := v.LastResponse.Results[rank].Raw["collection"].(string); ok {
+	if collection, ok := v.LastResponse.Results[rank].Raw["syscollection"].(string); ok {
```

Comment:
You meant to add the `sys` prefix again?
These are removed when using an elastic index.

Reply:
Yea.... but in the CloudV1 orgs the no sys prefix don't exist :( I'll look for both i guess to solve that issue.

Didn't know in elastic they didnt exist anymore.

Reply:
Maybe have `event.SourceName = getFromRaw("source")`; that will check `source` and `syssource`, that can be reused for `collection` and `urihash` (if need)